### PR TITLE
Clean up documentation

### PR DESCRIPTION
**Pull request by JafarAbdi**

### CODE REVIEW

File: doc/user.md [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+                  <param name="initial_value">0.0</param>
+                </state_interface>
+                <state_interface name="velocity"/>
+            </joint>
+        </ros2_control>
+```
+
+## Isaac Sim
+
+Running Isaac with ROS2 requires setting the RMW implementation to fastdds ([see](https://forums.developer.nvidia.com/t/issues-running-ros2-humble-with-isaac-2022-1-1/229173)) with the following config file [rtps_udp_profile.xml](https://github.com/NVIDIA-ISAAC-ROS/isaac_ros_common/blob/main/docker/middleware_profiles/rtps_udp_profile.xml).
```
Comment (MarqRazz) [inline doc/user.md:71]: This belongs in the beginning of the install section.

File: doc/user.md [hunk 2]
```diff
@@ -0,0 +1,78 @@
+# User Guide
+
+## Isaac System
```
Comment (MarqRazz) [inline doc/user.md:3]: Can we add instructions on how to run the example [Franka world](https://docs.omniverse.nvidia.com/app_isaacsim/app_isaacsim/tutorial_ros2_moveit.html)?

What command should the user run [here](https://docs.omniverse.nvidia.com/app_isaacsim/app_isaacsim/tutorial_ros2_moveit.html#running-moveit-2)? Does this depend on the `isaac_moveit` package? I though I remember you saying you can use the `moveit_resources` one.

File: doc/installation.md [hunk 3]
```diff
@@ -0,0 +1,48 @@
+# Installation
+
+## Binaries
+
+`isaac_ros2_control` is currently only released for ROS2 Humble and Rolling. For other ROS2 releases, you may need to build it from source.
```
Comment (MarqRazz) [inline doc/installation.md:5]: Did we really get this released?

File: doc/installation.md [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+
+        sudo apt-get install ros-$ROS_DISTRO-isaac-ros2-control
+
+## Build from Source
```
Comment (MarqRazz) [inline doc/installation.md:9]: FYI I'm going to make a follow up PR to build a docker container that users can run the demo against.

**Review by MarqRazz — APPROVED** [review]
Can't thank you enough for making `isaac_moveit.py`! I tested it against Isaac 1.1 and 2.0 and it worked great.

Anything else we need to do here @JafarAbdi?

**Comment by MarqRazz** [comment]
Sorry I pushed the launch file trying to get industrial_ci to clear the `clang-tidy` check but it didn't help. Can we set
```
- CI_NAME: clang-tidy
  CLANG_TIDY: false
```
in `build_and_test.yaml` to disable it?

File: doc/installation.md (hunk 3 above)
Comment (JafarAbdi) [reply doc/installation.md:5]: I was planning to get it released, I was hoping to do some testing before doing so

Comment (JafarAbdi) [reply doc/installation.md:5]: I'm more than happy to cut a release if you prefer that option

**Comment by JafarAbdi** [comment]
> Sorry I pushed the launch file trying to get industrial_ci to clear the `clang-tidy` check but it didn't help. Can we set
> 
> ```
> - CI_NAME: clang-tidy
>   CLANG_TIDY: false
> ```
> 
> in `build_and_test.yaml` to disable it?

I don't think that one will disappear unless someone with push access merges this PR, I think it's a bug in github :D

**Comment by MarqRazz** [comment]
So can I remove that requirement and merge this? Will that self correct once in main?

**Comment by JafarAbdi** [comment]
> So can I remove that requirement and merge this? Will that self correct once in main?

I already removed it. Yes, merging this will self resolve the issue on main

**Pull request merged**